README.md

```diff
@@ -254,7 +183,7 @@ class {'chocolatey':
 }
 ~~~
 
-### Set Chocolatey as Default Windows Provider
+#### Set Chocolatey as Default Windows Provider
```

Comment:
This probably goes elsewhere. It doesn't go under Managing Chocolatey's installation though.